### PR TITLE
Lyche different openinghours

### PR DESCRIPTION
**Pull request by Simonkv**

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -398,11 +398,22 @@
   add_index "standard_hours", ["area_id"], :name => "index_standard_hours_on_area_id"
   add_index "standard_hours", ["day"], :name => "index_standard_hours_on_day"
 
+  create_table "sulten_lyche_opening_hours", :force => true do |t|
+    t.time     "openLyche"
```
Comment (aleksanb) [inline db/schema.rb]: These should be snakecase values.

File: db/migrate/20160314224333_sulten_lyche_opening_hours.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class SultenLycheOpeningHours < ActiveRecord::Migration
+  def change
+    create_table :sulten_lyche_opening_hours do |t|
+      t.time :openLyche
```
Comment (aleksanb) [inline db/migrate/20160314224333_sulten_lyche_opening_hours.rb]: Again, use snake case fields 'open_lyche'.

File: app/controllers/sulten/lyche_opening_hours_controller.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    else
+      flash.now[:error] = "Oops, noe gikk galt."
+      redirect_to sulten_lyche_opening_hours_path
+    end
+  end
+
+  def create
+    @lyche_opening_hours = Sulten::LycheOpeningHours.new(params[:sulten_lyche_opening_hours])
+    if @lyche_opening_hours.save
+      flash[:success] = "Du lagde en ny apningsdag."
```
Comment (aleksanb) [inline app/controllers/sulten/lyche_opening_hours_controller.rb:24]: Translations perhaps?

File: app/controllers/sulten/lyche_opening_hours_controller.rb [hunk 4]
```diff
@@ -0,0 +1,31 @@
+class Sulten::LycheOpeningHoursController < ApplicationController
+  def index
+    @lyche_opening_hours= Sulten::LycheOpeningHours.order(:id).all
```
Comment (aleksanb) [inline app/controllers/sulten/lyche_opening_hours_controller.rb]: ID is the default ordering afaik.

File: app/controllers/sulten/lyche_opening_hours_controller.rb [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    @weekdays = ["Sondag","Mandag", "Tirsdag", "Onsdag", "Torsdag", "Fredag", "Lordag"]
```
Comment (aleksanb) [inline app/controllers/sulten/lyche_opening_hours_controller.rb]: Replace this with time.wday as in http://stackoverflow.com/questions/4844498/rails-3-finding-day-of-week.

**Comment by Simonkv** [comment]
Remember to run the rake task created in this branch before deploying.

**Comment by cristeahub** [comment]
Remember to rebase away my commit that says "Nå skal alt være klart, Simon" :)

**Comment by aleksanb** [comment]
Probably squash everything at a point.

**Comment by cristeahub** [comment]
> Probably squash everything at a point.

Yeah, that works too!

**Pull request closed without merging**